### PR TITLE
A notification signup reminder button for when the TA queue is empty

### LINKED ISSUE #301: TA notification signup reminder button
When TA's aren't registered for notifications, add a button below the "no questions in queue" message to remind TA's to sign up for notifications.

### PR DESCRIPTION
**Pull request by dfarooq610**
closes #301

### CODE REVIEW

File: packages/app/components/Queue/TA/TAQueueList.tsx
Comment (willstenzel) [inline packages/app/components/Queue/TA/TAQueueList.tsx:301]: Should we change this to check if only one of the notification types is enabled?
```suggestion
          {(!phoneNotifsEnabled && !desktopNotifsEnabled) && (
```

Don't know if everyone will choose to use both desktop and phone notifications. As long as they have one they should be good.

**Review by willstenzel — CHANGES_REQUESTED** [review]

**Comment by dajinchu** [comment]
let's get this to a good state, but I think we might want to wait until we fix the TA notifications and phone notifications before merging this?

**Review by willstenzel — APPROVED** [review]

File: packages/app/components/Queue/TA/TAQueueList.tsx
Comment (NEUDitao) [inline packages/app/components/Queue/TA/TAQueueList.tsx:266]: probably would be better named something like "signedUpForNotifications" or "hasSignedUpForNotifications" or something -- isNotifOpen is a little misleading, it implies that there's a notificaiton open on the screen

**Review by NEUDitao — APPROVED** [review]

File: packages/app/components/Queue/TA/TAQueueList.tsx
Comment (liustanley) [inline packages/app/components/Queue/TA/TAQueueList.tsx:276]: I would rename this styled component to `NotifRemindButton` since it's a styled Button

File: packages/app/components/Queue/TA/TAQueueList.tsx
Comment (liustanley) [inline packages/app/components/Queue/TA/TAQueueList.tsx:267]: I like that you included some styling with this button! For consistency, it would be nice if the button was styled the same as the "Edit Queue Details" button on the side: https://github.com/sandboxnu/office-hours/blob/283996ea793cba7fc69f83e645fd7a0a8e511839/packages/app/components/Queue/TA/TAQueueList.tsx#L77

**Review by liustanley — CHANGES_REQUESTED** [review]
clean code! two small changes and we good to merge

**Comment by NEUDitao** [comment]
@liustanley are we still blocked on this? it's been closed for 14 days

**Review by liustanley — APPROVED** [review]

**Pull request merged**
